virttest/migration_template.py

```diff
@@ -918,12 +919,31 @@ def cleanup(self):
         """
         Cleanup env
         """
-        logging.debug("Start to clean up env")
-        # Shutdown vms
+        logging.info("Start to clean up env")
+
+        # Remove vms on src host(if vm is migrated back)\
+        # or dest host(if vm is not migrated back)
+        logging.info("Remove vms on src or dest host")
         for vm in self.vms:
-            vm.destroy()
+            try:
+                vm.remove()
```

Comment:
if here we used the vm.remove, the vm should be undefined, why we need to undefine it again in line941?

Reply:
See explanation below